### PR TITLE
Passing the event argument to callbacks that use it. #88

### LINKED ISSUE #88: onMouseDown doesn't have the event parameter passed in correctly
This is the current onMouseDown event that is bound.

```
case 'onMouseDown':
{
  _this.editor.on('mousedown', function(cm) {
    _this.props.onMouseDown(_this.editor, event);
  });
  break;
}
```

The callback function doesn't have an event variable that is passed in - which causes **ReferenceError: event is not defined** in some browsers like Firefox.

### PR DESCRIPTION
**Pull request by anandsainath**
@scniro Pull request to fix #88

### CODE REVIEW

**Comment by anandsainath** [comment]
I'm assuming that updating version in the package.json file would automagically publish a new version. Would that be a good assumption?

**Comment by anandsainath** [comment]
Also, I'm not sure which check failed..

**Comment by anandsainath** [comment]
@scniro would be awesome if we could get this checked in soon! :)
I have a dependency on this for some other change that I'm tracking.

File: .npmignore [hunk 1]
```diff
@@ -9,3 +9,4 @@ docs
 src
 test
 gulpfile.js
+node_modules
```
Comment (scniro) [inline .npmignore]: @anandsainath what's going on with this? `node_modules` shouldn't be have an impact here

**Pull request merged**